tests/integration_tests/test_sklearn.py

```diff
@@ -5,12 +5,14 @@
 import pytest
 import scipy as sp
 from sklearn.datasets import make_blobs
+from sklearn.datasets import make_regression
 from sklearn.decomposition import PCA
 from sklearn.exceptions import ConvergenceWarning
 from sklearn.exceptions import NotFittedError
 from sklearn.linear_model import LogisticRegression
 from sklearn.linear_model import SGDClassifier
 from sklearn.neighbors import KernelDensity
+from sklearn.tree import DecisionTreeRegressor
 
 from optuna import distributions
 from optuna import integration
```

Comment:
Please remember to import `BruteForceSampler` if you accept the suggestion in `test_objective_error_score_nan_regression`.

```suggestion
from optuna import integration
from optuna.samplers import BruteForceSampler
```